src/lib.rs

```diff
@@ -0,0 +1,48 @@
+#[macro_use]
+extern crate indoc;
+extern crate serde_json;
+extern crate serde;
+use serde::ser;
+use serde_json::{Value, Error};
+
+pub fn something(pkg: &'static str) ->  Result<(), Error>
+{
+    // println!("{}", pkg);
+    // Parse the string of data into serde_json::Value.
+    let value: Value = serde_json::from_str(pkg)?;
+
+    // Access parts of the data by indexing with square brackets.
+    // println!("Please call {} at the number {}", value["name"], value["phones"][0]);
+    println!("-------------");
+    // println!("{}", serde_json::to_string(&value)?);
+    println!("-------------");
+    // println!("{}", serde_json::to_string_pretty(&value)?);
+
+    let mut writer = Vec::with_capacity(128);
+    let mut serializer = serde_json::Serializer::with_formatter(writer, serde_json::ser::PrettyFormatter::with_indent(b"  "));
+
+    value.serialize(&mut serializer);
```

Comment:
so as far as I can tell https://github.com/serde-rs/json/blob/master/src/ser.rs does actually call `value.serialize(&mut serializer);` but apparently I don't have the right value type, or don't have serialize in-scope:

error message for the above line:
```
no method named `serialize` found for type `serde_json::Value` in the current scope
```